packages/sdk-ethers/src/index.ts

```diff
@@ -10,28 +10,46 @@ export const erc6551RegistryAddress =
 export const erc6551AccountImplementationAddress =
   "0x36963236d915e4e9b5f70677eBD1ea3e69Cfbbd6" as const;
 
-export async function getAccount(
+export function getAccount(
   tokenContract: string,
   tokenId: string,
-  provider: providers.BaseProvider
-) {
-  const registry = new Contract(
-    erc6551RegistryAddress,
-    erc6551RegistryAbi,
-    provider
-  );
-
-  const { chainId } = await provider.getNetwork();
-
-  return registry.callStatic.account(
+  chainId: number,
```

Comment:
User either has a provider and can pass this in via `const { chainId } = await provider.getNetwork();` or they do not have a provider and can pass in whatever network they please.